<launch>
  <include file="$(env ROBOT_PKG)/launch/config.launch"/>


  <node
     machine="hrp2v"
     ns="/wide"
     pkg="stereo_image_proc"
     type="stereo_image_proc"
     name="stereo_image_proc_wide"
     respawn="true" >
    <param name="approximate_sync" value="True" />
  </node>

  <node
     machine="hrp2v"
     ns="/narrow"
     pkg="stereo_image_proc"
     type="stereo_image_proc"
     name="stereo_image_proc_narrow"
     respawn="true" >
    <param name="approximate_sync" value="True" />
    <param name="min_disparity" value="20" />
    <param name="disparity_range" value="128" />
    <param name="correletion_window_size" value="15" />
</node>

  <!-- optional visualization tools -->

  <group if="$(optenv ENABLE_CONSOLE 0)">
    <node
       pkg="rxtools"
       type="rxconsole"
       name="rxconsole" />
  </group>

</launch>
